templates/zerver/help/delete-a-stream.md

```diff
@@ -0,0 +1,22 @@
+# Delete a stream
+On Zulip, most of the stream administration is done on the subscription page. Nevertheless, that is not the case for deleting a stream.
+Follow this instructions to learn how it is done.
+
+## Locate the administration tab
+First of all you will need to be on an administrator account. To delete your stream, click the cog (![cog](/static/images/help/cog.png))
+at your top right corner. A dropdown menu with the administration tab will appear, also click this tab.
```

Comment:
Yeah, you can say something like that clicking the cog opens a menu, find the 'Administration' button in this menu and click it.